main_test.go

```diff
@@ -1976,7 +1981,7 @@ filter_policy {
 		require.Equal(t, http.StatusOK, w.Result().StatusCode)
 
 		responseBody := getResponseBody(t, w)
-		require.Contains(t, string(responseBody), "go_gc_duration_seconds")
+		require.Contains(t, string(responseBody), "promhttp_metric_handler_errors_total")
```

Comment:
Why this change?

Reply:
go collector and process collector setup are moved on main file, but in this test we setup the router. So, those metrics are not exposed. I change the test to check that the PolicyEvalDuration metric is exposed correctly